src/components/Spinner/Spinner.tsx

```diff
@@ -58,12 +59,13 @@ function Spinner({
   const spinnerSVG = size === 'large' ? spinnerLarge : spinnerSmall;
 
   return (
-    <svg
-      viewBox={spinnerSVG.viewBox}
-      dangerouslySetInnerHTML={{__html: spinnerSVG.body}}
+    <Image
+      alt="Spinner"
```

Comment:
I don't think we want this image to be announced to screenreaders. Set alt to an empty string: `alt=""`